### PR TITLE
Fix docs typos and improve grammar

### PR DESCRIPTION
**Pull request by akkspros**
Fix docs typos and improve grammar

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at support@wptide.org. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing or otherwise, unacceptable behavior may be reported by contacting the project team at support@wptide.org. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (jeffpaul) [inline CODE_OF_CONDUCT.md]: Please revert this change as the punctuation makes sense and it's inherited from the Contributor Covenant Code of Conduct and we'll only update if there's a change upstream.

File: docs/gcp/google-app-engine.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Google App Engine
 
-You need to setup App Engine before you can deploy to GCP or use GCP resources for local development.
+You need to setup an App Engine before you can deploy to GCP or use GCP resources for local development.
```
Comment (jeffpaul) [inline docs/gcp/google-app-engine.md]: Please revert this as the existing language makes sense.

File: docpress/docpress-base/example/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Example site
 
-Use this site to test the Docpress theme as you make changes. It's a simple metalsmith site; you can use `node metalsmith.js` or [metalsmith-start] (https://www.npmjs.com/package/metalsmith-start).
+Use this site to test the Docpress theme as you make changes. It's a simple metalsmith site; you can use `node metalsmith.js` or [metalsmith-start](https://www.npmjs.com/package/metalsmith-start).
 
-Make sure to `rm -rf cache/` first, and restart metalstart everytime you change Stylus files.
+Make sure to `rm -rf cache/` first, and restart metal start every time you change Stylus files.
```
Comment (jeffpaul) [inline docpress/docpress-base/example/README.md]: I think this is supposed to be metalsmith, please update to that

**Review by jeffpaul — CHANGES_REQUESTED** [review]
One more minor change and then this looks good to get merged in.

**Comment by akkspros** [comment]
@jeffpaul I have updated it when you have a chance, please check this.

**Review by jeffpaul — APPROVED** [review]
Looks good, thanks for updating!

**Pull request merged**